src/components/AddressEdit/AddressEdit.tsx

```diff
@@ -222,46 +222,48 @@ const AddressEdit: React.FC<AddressEditProps> = props => {
 
       <div className={classes.root}>
         <div>
-          <SingleAutocompleteSelectField
-            disabled={disabled}
-            autocomplete="new-password"
+          <Combobox
             data-test-id="address-edit-country-select-field"
-            displayValue={countryDisplayValue}
+            autoComplete="new-password"
```

Comment:
Suggestion: I remember this was a hack for browser keeping customers' addresses. Let's change this to "off" (there are multiple in this component)